### PR TITLE
Update stable Dockerfile to 1.18

### PR DESCRIPTION
**Pull request by suvl**
This PR is coming from PET-1284 and does four things and a half:

0. Updates alpine to 3.11
1. Updates the stable image to NGINX version 1.18
1.1 The use of geoip2 lead us to add and install the maxminddb lib
2. Updates NJS to 0.4.0
3. Adds the stream [STS plugin](https://github.com/vozlt/nginx-module-sts) to get metrics from stream servers.

### CODE REVIEW

File: stable/alpine/Dockerfile [hunk 1]
```diff
@@ -1,19 +1,23 @@
-FROM alpine:3.7
+FROM alpine:3.11
 
 LABEL maintainer="NOS Inovação <nosi.metadata@nos.pt>"
```
Comment (davidrneves) [inline stable/alpine/Dockerfile]: Should change the mantainer email to the platforms team

Comment (suvl) [reply stable/alpine/Dockerfile]: yap! great catch. done!

File: stable/alpine/Dockerfile [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
+LABEL maintainer="NOS Inovação <nosi.platforms@nos.pt>"
 
-ENV NGINX_VERSION 1.16.1
-ENV NJS_VERSION 0.3.5
+ENV NGINX_VERSION 1.18.0
+ENV NJS_VERSION 0.4.0
 
 # Ensure git
 RUN apk update && apk upgrade \
 	&& apk add --no-cache --virtual .build-deps bash git libmaxminddb-dev autoconf automake
```
Comment (ctorrao) [inline stable/alpine/Dockerfile:10]: The libmaxminddb was already in this statement, does it make sense to be here and compile directly from the source code below?

Comment (suvl) [reply stable/alpine/Dockerfile:10]: I thought I had removed that package...
The nginx build does not find the necessary libs from that package (maybe it's too old?) so I've opted to build it from source as it is quite simple and has no dependencies other than libc.

Comment (ctorrao) [reply stable/alpine/Dockerfile:10]: ok, fine by me :)

**Pull request merged**